### ISSUE TITLE
Drop using `Mix.Utils` in runtime.

### ISSUE THREAD

**Issue by KamilLelonek** [danielberkompas/ex_twilio#19]
Right now `Mix.Utils` module is being used in two places:

- https://github.com/danielberkompas/ex_twilio/blob/1752a0541ca3a1fd373e22fa8740a95ee9784970/lib/ex_twilio/url_generator.ex#L110
- https://github.com/danielberkompas/ex_twilio/blob/1752a0541ca3a1fd373e22fa8740a95ee9784970/lib/ex_twilio/url_generator.ex#L161

`Mix` is a build tool that isn't meant to be used in a runtime, but only for compilation purposes.

When building a production release of a project that uses `ex_twillio` developers are forced to add 

```elixir
defp apps do
  [
    :logger,
    :ex_twilio,
    :mix,
  ]
end
```

which should not happen.

**Comment by danielberkompas** [comment]
Good point. I'll need to create camelize and underscore functions, or else drop in a dependency to provide those.

**Comment by danielberkompas** [comment]
This has been fixed, and the fix is included in the new `0.1.3` release.
